molo/commenting/models.py

```diff
@@ -98,3 +105,27 @@ class Meta:
         verbose_name_plural = 'Canned responses'
         app_label = 'commenting'
         ordering = ['response_header', 'response']
+
+
+@register_setting
+class CommentingSettings(BaseSetting):
+    """
+
+    This class will allow content creator to add an anonymous alias.
+    This is for users who want to comment anonymously.
+    The alias is added per site.
+    """
+
+    commenting_anonymous = models.TextField(
+        verbose_name='Commenting Anonymous Alias',
+        default="Anonymous",
+        help_text="This is the name used when users comment anonymously"
```

Comment:
this help text looks a bit funky, can we be more clear about which user e.g front-end user or admin user